### DISCUSSION TITLE
Font ?

### DISCUSSION THREAD

**Discussion by CoderZ90** [olimorris/onedarkpro.nvim#194] [Q&A]
Hey 👋 great work 
But wanted to ask which font you are using in your screenshots
Thanks in advance 😃

**Comment by olimorris** [comment]
[Operator Mono](https://www.typography.com/blog/introducing-operator). It's not cheap but is splendid. I've configured it [here](https://github.com/olimorris/dotfiles/blob/main/.config/kitty/kitty.conf).
